docker-compose.yml

```diff
@@ -1,10 +1,15 @@
 version: '2'
 
+volumes:
+  pg_data:
+
 services:
 
   api_db:
     container_name: api_db
     image: mdillon/postgis:11-alpine
+    volumes:
+      - pg_data:/var/lib/postgresql/data
```

Comment:
Not sure I understand this... Is this the default location for a linux postgres install, and if so wouldn't it potentially step on whatever else a developer might have there?

Reply:
Should mount the container directory (and contents) outside in the host